src/nexuscli/api/repository/model.py

```diff
@@ -418,6 +418,20 @@ def __init__(self, name, depth=1, **kwargs):
 
         super().__init__(name, **kwargs)
 
+    @property
+    def configuration(self):
+        """
+        As per :py:obj:`Repository.configuration` but specific to this
+        repository recipe and type.
+
+        :rtype: str
+        """
+        repo_config = super().configuration
+        repo_config["attributes"]["yum"] = {
```

Comment:
nitpick: the rest of the codebase primarily uses single quotes.
```suggestion
        repo_config['attributes']['yum'] = {
```